.github/workflows/update_dependencies.yml

```diff
@@ -0,0 +1,68 @@
+name: Update dependencies
+on:
+  schedule:
+    # Run automatically at 6AM PST Tuesday
+    - cron: '0 6 * * 2'
+  workflow_dispatch:
+
+jobs:
+  update_dependencies_and_test:
```

Comment:
nit: `kebab-case` seems more standard. Should update the job names. Same for the filename.